.hound.yml

```diff
@@ -8,3 +8,6 @@ scss:
 
 jshint:
   config_file: .jshintrc
+
+haml:
+  config_file: .haml.yml
```

Comment:
Wouldn't `.hound.haml.yml` make more sense?  But can be fixed in a follow-up if necessary.